operator/pkg/ingress/annotations/annotations.go

```diff
@@ -24,9 +25,10 @@ const (
 	defaultTCPKeepAliveInitialIdle   = 10 // in seconds
 	defaultTCPKeepAliveProbeInterval = 5  // in seconds
 	defaultTCPKeepAliveMaxProbeCount = 10
+	defaultWebsocketsEnabled         = 1 // 1 - Enabled, 0 - Disabled
```

Comment:
the default value could be disabled

Reply:
Having it disabled breaks a bunch of end user applications by default. Is it okay to have it enabled so that Cilium ingress works the same as nginx and traefik by default?

Reply:
That said, I've gone ahead and made this change anyway.

Reply:
good question, I think it's a mix between different ingress controllers.  AFAIK, contour has WebSocket disabled by default.